testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -1135,23 +1135,32 @@ public void testCreateSchema()
     @Test
     public void testDropNonEmptySchema()
     {
+        String schemaName = "test_drop_non_empty_schema_" + randomTableSuffix();
         if (!supportsCreateSchema()) {
-            assertQueryFails("DROP SCHEMA " + getSession().getSchema().orElseThrow(), "This connector does not support dropping schemas");
+            assertQueryFails("CREATE SCHEMA " + schemaName, "This connector does not support creating schemas");
```

Comment:
Revert.

Reply:
This will not test whether connector supports dropping schema as dropping non-empty schema will throw first, before calling metadata.dropSchema

Reply:
It's intentional here.

Reply:
From whitebox testing perspective -- i agree.
Then you should just `return;` here, with a comment.
`assertQueryFails("CREATE SCHEMA` is redundant wrt `testCreateSchema`